courses/admin.py

```diff
@@ -377,6 +375,14 @@ class CourseTopicAdmin(ModelAdmin, admin.ModelAdmin):
     delete_view_class = TopicsWagtailDeleteView
 
 
+class PartnerAdmin(admin.ModelAdmin):
```

Comment:
I think this should inherit from `TimestampedModelAdmin` to reuse its display properties, I could not see date fields on the partner detail page in Django Admin.
<img width="1473" alt="image" src="https://github.com/mitodl/mitxpro/assets/34372316/2c09417e-3b40-499e-89c9-c4eb825c8559">